tests/eko/test_kernels_s.py

```diff
@@ -149,6 +149,52 @@ def test_zero_nnlo_decompose(monkeypatch):
             pass
 
 
+def test_zero_n3lo_decompose(monkeypatch):
```

Comment:
we get a new warning here as well
```
tests/eko/test_kernels_s.py::test_gamma_usage
  /home/felix/Physik/N3PDF/EKO/eko/src/eko/anomalous_dimensions/__init__.py:69: RuntimeWarning: invalid value encountered in cdouble_scalars
    c = 1.0 / det
```
as before: please explain (and suppress) or avoid